src/attributes.jl

```diff
@@ -610,14 +618,15 @@ An optimizer attribute for the `Vector{AbstractOptimizerAttribute}` of all optim
 struct ListOfOptimizerAttributesSet <: AbstractOptimizerAttribute end
 
 """
-    SolverName()
+    SolverName()::String
```

Comment:
This way of documenting the return value of `return_type(::SolverName)` is confusing; it reads like the constructor for `SolverName` returns a String.